releasenotes/notes/octavia-alerts-a7f1fe041f5450be.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Adds alerts for Octavia load balancers and amphorae. Alerts are triggered when
```

Comment:
It would be good to configure your editor to wrap the reno lines at 78 characters. Upstream linters run doc8 which would fail on them.

Reply:
doc8 actually sets the limit at 79 characters.